File: springdoc-openapi-starter-webflux-api/src/test/java/test/org/springdoc/api/app85/entity/Person.java
package test.org.springdoc.api.app85.entity;

public class Person {

	private String id;

	private Sex sex;
	private String firstName;
	private String lastName;
	private String age;
	private String interests;
	private String email;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public Sex getSex() {
		return sex;
	}

	public void setSex(Sex sex) {
		this.sex = sex;
	}

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getAge() {
		return age;
	}

	public void setAge(String age) {
		this.age = age;
	}

	public String getInterests() {
		return interests;
	}

	public void setInterests(String interests) {
		this.interests = interests;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}
}

File: springdoc-openapi-tests/springdoc-openapi-kotlin-data-rest-tests/src/test/kotlin/test/org/springdoc/api/v31/app1/Person.kt
package test.org.springdoc.api.v31.app1

import jakarta.persistence.Embeddable
import jakarta.persistence.Embedded
import jakarta.persistence.Entity
import jakarta.persistence.Id

@Entity
class Person {

	@Id
	var id: Long? = null

	@Embedded
	var name: Name? = null
}

@Embeddable
class Name {

	var firstName: String? = null

	var lastName: String? = null
}
